wallet/wallet.go

```diff
@@ -502,3 +502,15 @@ func (w *Wallet) SignMessage(password, addr, msg string) (string, error) {
 
 	return prv.Sign([]byte(msg)).String(), nil
 }
+
+func (w *Wallet) GetVersion() int {
```

Comment:
Just a note: In our codebase, we typically avoid using `Get` as a prefix for getter methods. Let's keep our naming consistent with this convention.

Reply:
Fixed. I also renamed `GetHistory` function that was a bit below this function

Reply:
Thanks